memcached/README.md

```diff
@@ -0,0 +1,45 @@
+# YCSB Memcached binding
```

Comment:
I was under the impression all files needed licensing. For .md files you have to look at the raw to see the licensing in the comments. Here's the example from mongodb again: https://raw.githubusercontent.com/brianfrankcooper/YCSB/master/mongodb/README.md